### DISCUSSION TITLE
Tighter Appearance

### DISCUSSION THREAD

**Discussion by og900aero** [gokcehan/lf#1635] [Ideas]
Would be great if the appearance will be tighter.

This is lf:
![screenshot_1710160020](https://github.com/gokcehan/lf/assets/49487376/96e0d168-6c74-4223-b516-795d8c18f574)

And better appearance:
![screenshot_1710160920](https://github.com/gokcehan/lf/assets/49487376/1c78f6a6-e5fb-4a57-ba75-92cc8bb093cd)

**Comment by DusanLesan** [comment]
What exactly is the discussion about? You want more compact UI? I do not see much difference in space taken except that "better" version has less padding on  the left side

**Reply by og900aero** [reply]
Yes, compact ui. Better use of space.

**Comment by joelim-work** [comment]
`lf` reserves two cells to the left of the filename. The first cell is for the selection marker (press <kbd>space</kbd>), and the second cell is for tags (press <kbd>t</kbd>).

**Reply by og900aero** [reply]
Ohh, I understand. And I don't think it can be solved that both use the same cell, because although one uses the color of the background, the other puts a star in the foreground, but since the appearance of both can be changed, it could be a big mess. Am I Right?

**Reply by joelim-work** [reply]
Merging selection and tags into a single cell would be potentially confusing, and I suspect it would cause issues anyway, for example if a user wanted to set both the foreground and background colors for tags:

```
set tagfmt "\033[31;42m"
```

**Reply by og900aero** [reply]
Yeah, that's what I thought too. Then, unfortunately, it will stay that way.

**Comment by CatsDeservePets** [comment]
Hello @og900aero!
Just so you know, I just merged a setting for this in https://github.com/gokcehan/lf/pull/2330.

**Reply by og900aero** [reply]
Thx for the commit. I tried it, but I stayed old version, because is too tight:

<img width="173" height="55" alt="image" src="https://github.com/user-attachments/assets/3eb0ac21-0da0-404a-834a-46a9c4995032" />


Better this version of yazi:

<img width="189" height="63" alt="image" src="https://github.com/user-attachments/assets/ac1941be-7cc5-428c-b0ca-8d734afe1a36" />

**Reply by CatsDeservePets** [reply]
I am sorry to hear that.

**Reply by joelim-work** [reply]
The `mergeindicators` option is quite similar to the 'choose between regular or compact view' UI setting found in various applications. It is high-level so users can configure it easily, but ultimately not flexible which means not every user can be satisfied.

On the other hand the UI in `yazi` can be scripted in Lua so virtually anything is possible (with some work).
